### ISSUE TITLE
"Hosts from filter" and "Services from filter"does not work

### ISSUE THREAD

**Issue by Gian1110** [Icinga/icingaweb2-module-businessprocess#364]
## I'm trying to create a business process with "hosts from filter" and "Services from filter" But it doesn't work. On the first image I don't see any host though I have 9 hosts. On the second image it doesn't even show the filter

![filterhost](https://user-images.githubusercontent.com/44478908/201176980-080a957f-f530-408e-8b6f-685daaf9bbde.jpeg)
![filterservice](https://user-images.githubusercontent.com/44478908/201177015-7f54d054-f352-4377-8da9-2a587de988e8.jpeg)
## I’m running icinga2 dockerized with the jordan/icinga2 image

**Comment by lippserd** [comment]
Hi,

You are filtering for host `App` (exact match) and for service `ping4` (exact match). Is that correct?

All the best,
Eric

**Comment by Gian1110** [comment]
yes, it's true. At First, i filter by host (exact match the host name), then by service (exact match the service name)

**Comment by lippserd** [comment]
> yes, it's true. At First, i filter by host (exact match the host name), then by service (exact match the service name)

Do you have a host called `App`, i.e. `object Host "App" { ... }`? `host_name` does not filter for `display_name`.

**Comment by Gian1110** [comment]
> > yes, it's true. At First, i filter by host (exact match the host name), then by service (exact match the service name)
> 
> Do you have a host called `App`, i.e. `object Host "App" { ... }`? `host_name` does not filter for `display_name`.

yes, i have a host called "App" and display_name is equal to host_name.
I am trying with Display_name but dont work.

**Comment by nilmerg** [comment]
Sounds like #349. Oh, and it's `service_description`, not `service_name`.

**Comment by Gian1110** [comment]
> Sounds like #349. Oh, and it's `service_description`, not `service_name`.

thanks.
Now the filters work but they don´t show any host/service.
I am writing host name and services name exactly as i have
